rebar.config

```diff
@@ -4,11 +4,11 @@
 
 {xref_checks,
   [undefined_function_calls,undefined_functions,
-    locals_not_used,
     deprecated_function_calls, deprecated_functions]}.
 
 {cover_excl_mods,
-  [testutil,
+  [leveled_filterlexer, leveled_filterparser, leveled_evallexer, leveled_evalparser,
```

Comment:
What is the reason to exclude parser and lexer from coverage?
When I ran quickcheck property, I cover all code apart from some error cases. Was adding needless negative tests the reason?

Reply:
It is the auto-generated files that were the problem.  The eunit/ct tests cover leveled_filter, and leveled_eval modules - but there was a large number of coverage holes in the auto-generated files.

Seen as they are auto-generated, I din't think they were relevant to coverage.